industrial_ci/src/tests/source_tests.sh

```diff
@@ -97,6 +97,33 @@ function run_clang_tidy_check {
     fi
 }
 
+function run_pylint_check {
+    local target_ws=$1
+    local status=0
+    local exit_code=0
+
+    local -a pylint_versions
+    ici_parse_env_array pylint_versions PYLINT_VERSIONS
+    local -a pylint_args
+    ici_parse_env_array pylint_args PYLINT_ARGS
+
+    for cmd in "${pylint_versions[@]}"; do
+        ici_run "install_$cmd" ici_install_pkgs_for_command "$cmd" "$cmd"
```

Comment:
Do not show the install process by default:
```suggestion
        ici_quiet ici_install_pkgs_for_command "$cmd" "$cmd"
```
Or split these loops.